Dockerfile

```
@@ -1,12 +1,21 @@
 
 COPY ./server/go.mod ./server/go.sum ./
 RUN go mod download && go mod verify
 
 COPY ./server/. ./
-COPY ./dist/. ./static/
-RUN go build -v -o /usr/local/bin/app ./...
+RUN CGO_ENABLED=0 go build -v -ldflags="-s -w" -o app ./...
 
+# Runtime stage
+FROM alpine:3.16
+RUN apk --no-cache add ca-certificates
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)